modules/va_forms/app/sidekiq/va_forms/form_builder.rb

```diff
@@ -8,91 +8,124 @@ class FormBuilder
     include Sidekiq::Job
     include SentryLogging
 
-    # @current_retry set by Sidekiq middleware
-    attr_accessor :current_retry
-
-    RETRIES = 7 # Try 7 times over ~46 minutes
-    FORM_BASE_URL = 'https://www.va.gov'
+    FORM_FETCH_ERROR_MESSAGE = 'The form could not be fetched from the url provided.'
     STATSD_KEY_PREFIX = 'api.va_forms.form_builder'
 
-    sidekiq_options(retries: RETRIES)
+    sidekiq_options retry: 7
+
+    sidekiq_retries_exhausted do |msg, error|
+      args = msg['args'] || {}
+      form_name = args['fieldVaFormNumber']
+      row_id = args['fieldVaFormRowId']
+
+      # Ensure that the form is marked "valid_pdf: false" if successive form fetches have failed
+      if error.message == FORM_FETCH_ERROR_MESSAGE
+        form = VAForms::Form.find_by(row_id:)
+        url = VAForms::Form.normalized_form_url(args.dig('fieldVaFormUrl', 'uri'))
+        form_previously_valid = form&.valid_pdf
+        form&.update!(valid_pdf: false, sha256: nil, url:)
+        if form_previously_valid
+          VAForms::Slack::Messenger.new(
+            {
+              class: self.class.name,
+              message: "URL for form_name: #{form_name}, row_id: #{row_id} no longer returns a valid PDF or web page.",
+              form_url: url
+            }
+          ).notify!
+        end
+      end
+
+      # Log error and increment StatsD metric
+      message = "#{self.class.name} failed to import form_name: #{form_name}, row_id: #{row_id} into forms database."
+      Rails.logger.error(message, error.message)
+      StatsD.increment("#{STATSD_KEY_PREFIX}.failure", tags: { form_name:, row_id: })
+    end
 
     def perform(form_data)
       build_and_save_form(form_data)
-    rescue => e
-      form_name = form_data['fieldVaFormNumber']
-      message = "#{self.class.name} failed to import #{form_name} into forms database"
-      Rails.logger.error(message, e)
-      log_message_to_sentry(message, :error, body: e.message)
-      StatsD.increment("#{STATSD_KEY_PREFIX}.failure", tags: { form_name: })
-      raise
```

Comment:
The errors being logged here were too noisy to be very useful. Now, the specific exception will be raised and an error message will be logged to the Rails console when the job exhausts its retries. The `StatsD` counter will only be incremented when the retries are exhausted.